src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -172,7 +172,7 @@ internal static int Encrypt(IntPtr handlePtr, IntPtr buffer, int offset, int cou
             {
                 if (capacityNeeded > bufferCapacity)
                 {
-                    throw CreateSslException("OpenSsl::Encrypt outBufferPtr should be null");
+                    throw CreateSslException("OpenSsl::Encrypt capacity needed is more than buffer capacity. capacityNeeded = " + capacityNeeded + "," + "bufferCapacity = " + bufferCapacity);
```

Comment:
@rajansingh10 is aware of this and is tracking it for a subsequent PR via https://github.com/dotnet/corefx/issues/3362.